nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseCell.java

```diff
@@ -82,8 +84,19 @@ public void process(final InputStream in) throws IOException {
             }
         });
 
-        final Collection<PutColumn> columns = Collections.singletonList(new PutColumn(columnFamily, columnQualifier, buffer));
-        return new PutFlowFile(tableName, row, columns, flowFile);
+
+        final Collection<PutColumn> columns = Collections.singletonList(new PutColumn(columnFamily.getBytes(StandardCharsets.UTF_8),
+                                                                            columnQualifier.getBytes(StandardCharsets.UTF_8), buffer));
+
+        //check to see if we need to modify the rowKey before we pass it down to the PutFlowFile
+        byte[] rowKeyBytes = null;
```

Comment:
Might be able to create a utility method in the abstract class like `byte[] getRow(String row, String encoding)` since it looks PutHBaseCell and PutHBaseJson both need the same logic